packages/agent/conf/demos/knb/kvk/templates/kvk/KvkInschrijving.hbs

```diff
@@ -6,9 +6,9 @@
         "VerifiableCredential",
         "KvkInschrijving"
     ],
-    "expirationDate": "{{{ dateTimeAfterMonths 12 }}}"
+    "expirationDate": "{{{ dateTimeAfterMonths 12 }}}",
     "credentialSubject": {
-        "Omschrijving": "KvK inschrijving",
+        "description": "Company registered with the KvK",
```

Comment:
is this the correct language?

Reply:
Maarten request Dutch for this demo

Reply:
I reverted this along with Voornaam -> firstName
Actually this would be a correct i18n way:

{
    "@context": [
        "[https://www.w3.org/2018/credentials/v1",](https://www.w3.org/2018/credentials/v1%22,) {
            "description": {
                "@id": "[http://schema.org/description",](http://schema.org/description%22,)
                "@container": "@language"
            },
        }
    ],
    "type": [
        "VerifiableCredential",
        "NotaryCompanyVerification"
    ],
    "expirationDate": "{{{ dateTimeAfterMonths 12 }}}",
    "credentialSubject": {
        "description": {
          "en": "Company has been verified by the notary",
          "nl": "Bedrijf is geverifieerd door de notaris"
         }
        {{{mergeJSON @root}}}
    }
}

Does our wallet support that? Do we just dump the credentialSubject 1:1 or do we actually read from it?